<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::CodecBrowser</class>
 <widget class="QWidget" name="GammaRay::CodecBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <widget class="QSplitter" name="splitter_11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeView" name="codecList"/>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <item>
        <widget class="QLineEdit" name="codecText">
         <property name="placeholderText">
          <string>codec preview text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="selectedCodecs"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
